src/main/java/com/ericsson/ei/subscriptionhandler/SubscriptionValidator.java

```diff
@@ -16,21 +16,30 @@
 */
 package com.ericsson.ei.subscriptionhandler;
 
-import com.ericsson.ei.controller.model.NotificationMessageKeyValue;
-import com.ericsson.ei.controller.model.Subscription;
-import com.ericsson.ei.exception.SubscriptionValidationException;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 import org.apache.commons.lang3.StringUtils;
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.ValidationException;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.json.JSONTokener;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.http.MediaType;
 
-import java.util.List;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
+import com.ericsson.ei.controller.model.NotificationMessageKeyValue;
+import com.ericsson.ei.controller.model.Subscription;
+import com.ericsson.ei.exception.SubscriptionValidationException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 public class SubscriptionValidator {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SubscriptionValidator.class);
+    private static final String SCHEMA_FILE_PATH = "/schemas/subscription_schema.json";
```

Comment:
please make all the methods static in this class.

Reply:
done